src/rust/engine/Cargo.toml

```diff
@@ -24,7 +24,7 @@ futures-cpupool = { git = "https://github.com/alexcrichton/futures-rs", branch =
 glob = "0.2.11"
 ignore = "0.1.7"
```

Comment:
per Cargo.lock, looks like this got bumped to 0.1.8 somehow - so probably explicitly bump that too while you're in here?